openc3-cmd-tlm-api/app/controllers/auth_controller.rb

```diff
@@ -20,31 +20,30 @@
 require 'openc3'
 require 'openc3/models/auth_model'
 
-# This controller only exists in base since Keycloak handles auth in EE
-begin
-  require 'openc3-enterprise/controllers/auth_controller'
```

Comment:
This is needed, unless you are handling somehow different for enterprise.

Reply:
I don't think this file exists in Enterprise